lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -246,7 +246,9 @@ def check_arguments_for_rescoring(usage_key):
     corresponding module doesn't support rescoring calls.
     """
     descriptor = modulestore().get_item(usage_key)
-    if not hasattr(descriptor, 'module_class') or not hasattr(descriptor.module_class, 'rescore_problem'):
+    if (
+            not hasattr(descriptor, 'module_class') or not hasattr(descriptor.module_class, 'rescore_problem')
+    ) and not hasattr(descriptor, 'rescore'):
```

Comment:
I think we want to find rescore on the module_class instead of the descriptor (since rescoring needs the user bound to the module).  But that's an xmodule distinction, not relevant to XBlocks.  Can @nasthagiri weigh in here? Is this pattern going to cause problems for XModules like Capa, when they start using ScorableXBlockMixins?